Psyche/Assets/Scripts/UI/IntroController.cs

```diff
@@ -8,11 +8,20 @@
 /// </summary>
 /// Author: jmolive8, dnguye99
 public class IntroController : MonoBehaviour
-{
-    public GameObject intro1, intro2, intro3, intro4, intro5, intro6, intro7;
-    private Image img1, img2, img3, img4, img5, img6, img7;
+{   
+    [Header("Images")]
+    public GameObject Intro1;
+    public GameObject Intro2;
+    public GameObject Intro3;
+    public GameObject Intro4;
+    public GameObject Intro5;
+    public GameObject Intro6;
+    public GameObject Intro7;
+
+    //variables
+    private Image Img1, Img2, Img3, Img4, Img5, Img6, Img7;
     private bool IsTransitioning;
-    int curScreen = 1;
+    int CurScreen = 1;
```

Comment:
_private_ here too.

Reply:
Corrected in commit [here](https://github.com/MissionToPsyche/tungsten_12a_web_game-se/pull/253/commits/9e0100f194fd60ed48e6f56ae8c83340000e2dde) as well